### PR TITLE
add wasm documentation

### PR DESCRIPTION
**Pull request by moAlobaidi**
This PR partially addresses this [issue](https://github.com/AleoHQ/sdk/issues/626) and traces this aleo-docs [PR](https://github.com/AleoHQ/aleo-docs/pull/233)

### CODE REVIEW

File: documentation/sdk/sdk/00_sdk_overview.md [hunk 1]
```diff
@@ -23,13 +23,9 @@ Happy hacking!
 2. To regenerate the documentation, run `npx jsdoc --configure jsdoc.json --verbose`
 
 ## Aleo Tools
-<!-- markdown-link-check-disable -->
 
-[Aleo SDK account generator](https://aleo.tools/)
+[Aleo SDK account generator](https://aleohq.github.io/aleo/)
```
Comment (moAlobaidi) [inline documentation/sdk/sdk/00_sdk_overview.md:27]: This should be fixed in a PR pointed at the sdk repo as these links will continue to break anytime the documentation is generated. I will open an issue to address this.

Comment (collinc97) [reply documentation/sdk/sdk/00_sdk_overview.md:27]: @moAlobaidi can you open a PR to fix it directly?

**Review by collinc97 — COMMENTED** [review]
A few of the sdk documentation files now have nested `sdk/sdk/` directories is this intended?

**Comment by moAlobaidi** [comment]
> A few of the sdk documentation files now have nested `sdk/sdk/` directories is this intended?

Hmm might need some more clarification here. The new intended structure is:

-documentation
--- sdk
------ sdk
--------- [...documentation]
------ wasm
--------- [...documentation]

**Comment by collinc97** [comment]
It would be more intuitive to name the directory `sdk/typescript/` in my opinion to better align with `sdk/wasm/`.
Is that possible? - I realize this is the structure that is pulled from the SDK github repo.

**Comment by moAlobaidi** [comment]
Yea of course, will do!

**Review by collinc97 — APPROVED** [review]
LGTM

**Pull request merged**